src/main/java/lottoGame/model/winner/BeforeWinNums.java

```diff
@@ -0,0 +1,16 @@
+package lottoGame.model.winner;
+
+import java.util.Collections;
+import java.util.List;
+
+public class BeforeWinNums {
+    private final List<Integer> winNums;
```

Comment:
```suggestion
    private final List<LottoNum> winNums;
```
LottoNum을 사용하지 않은 이유는?